buildSrc/src/main/kotlin/mockito.publication-conventions.gradle.kts

```diff
@@ -0,0 +1,164 @@
+import gradle.kotlin.dsl.accessors._2a0859b2b1624a8c1f98b2dbe4a3b98d.javadoc
```

Comment:
I am not that familiar with Kotlin Gradle scripts yet, but to me this looks like an internal import that we should not be using. Is this intentional?

Reply:
Thank you it shouldn't be here !